src/Lister.php

```diff
@@ -33,15 +37,31 @@ public function renderView()
             throw new Exception(['Lister requires you to specify template explicitly']);
         }
         $this->t_row->trySet('_id', $this->name);
-        //$this->t_totals = isset($this->template['totals']) ? $this->template->cloneRegion('totals') : null;
+        $rowHTML = '';
 
         foreach ($this->model as $this->current_id => $this->current_row) {
             if ($this->hook('beforeRow') === false) {
                 continue;
             }
 
-            $rowHTML = $this->t_row->set($this->current_row)->render();
-            $this->template->appendHTML('rows', $rowHTML);
+            $this->t_row->trySet('_title', $this->model[$this->model->title_field]);
+            $this->t_row->trySet('_href', $this->app->url(['id'=>$this->current_id]));
+            $this->t_row->trySet('_id', $this->current_id);
+
+            if ($this->t_row == $this->template) {
```

Comment:
I slightly don't like this condition to be executed on every loop. That can have bad performancevespeciallyif templates are longer. What if we normalize template in `initChunks()` - if template don't have `{rows}{row}` tags, then add them and further work with only them in `renderView()`?

Reply:
i actually had some problems with it. Don't, mind if that's in a separate PR. Performance wise, it's just comparison, shouldn't be too bad.